src/components/Modal.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Modal = ({title, children, onExitClick}) => {
+  return (
+    <div>
+      <div class="modal is-active">
+        <div onClick={onExitClick} class="modal-background"></div>
+        <div class="modal-card">
+          <header class="modal-card-head">
+            <p class="modal-card-title">{title}</p>
+          </header>
+          <section class="modal-card-body">
+            {children}
+          </section>
+        </div>
+        <button
+          class="modal-close is-large"
+          aria-label="close"
+          onClick={onExitClick}
+        ></button>
```

Comment:
Is there a way to get the close button onto the modal itself? Probably more intuitive for users